introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/README.md

```diff
@@ -0,0 +1,2 @@
+### SageMaker JumpStart Sentence Pair Classification Training & Deployment
+This notebook `Amazon_JumpStart_Sentence_Pair_Classification.ipynb` for  Sentence Pair Classification task demonstrates (a) how to use a pre-trained model available in JumpStart for inference, and (b) how to fin-etune a pre-trained Transformer model on a custom dataset using JumpStart transfer learning algorithm, and how to use the fine-tuned model for inference.
```

Comment:
Typo: `fin-etune` -> `fine-tune`, extra space in `for  Sentence`

Reply:
fixed.